src/sass/themes/_themes.scss

```diff
@@ -0,0 +1,21 @@
+@use "../abstracts/variables.scss" as *;
+
+:root:not([dark]) {
+  background-color: $color-gray-light;
+  background-image: url($url-background-light-image);
+  $color-result-card-bg: $color-yellow-base !global;
+  $color-result-card-bg-active: $color-yellow-active !global;
+  $color-result-card-txt: $color-brown-base;
+  $box-shadow-result-card: $box-shadow-yellow;
+  $box-shadow-result-card-hover: $box-shadow-yellow-hover;
+}
+
+:root[dark] {
+  background-color: $color-gray-dark;
+  background-image: url($url-background-dark-image);
+  $color-result-card-bg: $color-brown-base !global;
+  $color-result-card-bg-active: $color-brown-active !global;
+  $color-result-card-txt: $color-yellow-base !global;
+  $box-shadow-result-card: $box-shadow-brown !global;
+  $box-shadow-result-card-hover: $box-shadow-brown-hover !global;
+}
```

Comment:
These are not css variables.
Read the basics https://developer.mozilla.org/en-US/docs/Web/CSS/Using_CSS_custom_properties

they have to be prefixed with `--`, so for example `--box-shadow-result-card-hover`